serialize.h

```diff
@@ -50,21 +50,22 @@ int serialize_header(char *buf, int bufsize, struct CcpMsgHeader *hdr);
 #define  UPDATE_FIELDS 3
 
 // Some messages contain strings.
-#define  BIGGEST_MSG_SIZE  1024
+#define  BIGGEST_MSG_SIZE  32000
 
 // for create messages, we know they are smaller when we send them up
 #define CREATE_MSG_SIZE     512
-// size of report msg is 12 B * MAX_REPORT_REG
+// size of report msg is approx MAX_REPORT_REG * 8 + 4 + 4
 #define REPORT_MSG_SIZE     900
 
 // Some messages contain serialized fold instructions.
-#define MAX_EXPRESSIONS    10 // 10 * 4 = 40 bytes for expressions
-#define MAX_INSTRUCTIONS   50 // 50 * 16 = 800 bytes for instructions
-#define MAX_IMPLICIT_REG   6
-#define MAX_REPORT_REG     15 // measure msg is 15*8 + 4 = 124 bytes
-#define MAX_CONTROL_REG    15
+#define MAX_EXPRESSIONS    50 // 10 * 4 = 40 bytes for expressions
+#define MAX_INSTRUCTIONS   300 // 50 * 16 = 800 bytes for instructions
+#define MAX_IMPLICIT_REG   6  // fixed number of implicit registers
+#define MAX_REPORT_REG     100 // measure msg 100 * 8 + 4 + 4
+#define MAX_CONTROL_REG    100 // arbitrary
```

Comment:
can we make this exactly what it supports instead of something arbitrarily lower? no reason to have it 100 if we can support e.g. 110 or something